media.css

```diff
@@ -0,0 +1,1400 @@
+    background: linear-gradient(2.6deg, #000 92.7%, #fff 93%);
+    padding-bottom: 110px;
+    padding-top: 100px;
+    margin-bottom: 0;
+  }
+
+  .video-section__image {
+    display: flex;
+    justify-content: center;
+    order: 1;
+  }
+
+  .video {
+    display: none;
+  }
+
+  .video-adaptive {
+    display: block;
+    width: 90%;
+    height: auto;
+    margin: 0 auto;
+  }
+
+  .video-section__content {
+    display: flex;
+    flex-direction: column;
+    order: -1;
+    width: 88%;
+    align-items: center;
+  }
+
+  .video-content__h3 {
+    width: 100%;
+    font-size: 30px;
+    line-height: 37px;
+    margin-bottom: 15px;
+    text-align: center;
+  }
+
+  .video-content__description {
+    width: 100%;
+    font-size: 20px;
+    line-height: 25px;
+    margin-bottom: 35px;
+    text-align: center;
+  }
+
+  .stories-content {
+    width: 100%;
+    background-color: #000;
+    color: #fff;
+    margin-top: -5px;
+    margin-bottom: 0;
+    padding-bottom: 130px;
+  }
+
+  .stories-content__h4 {
+    font-size: 30px;
+    line-height: 29px;
+    text-align: center;
+    margin-bottom: 45px;
+    color: #fff;
+  }
+
+  .stories-items {
+    width: 88%;
+    margin: 0 auto;
+  }
+
+  .stories-items__item {
+    width: 46%;
+  }
+
+  .stories-item__image {
+    display: block;
+    width: 100%;
+    margin: 0 auto;
+    margin-bottom: 33px;
+  }
+
+  .stories-item__title {
+    font-size: 25px;
+    line-height: 24px;
+    margin-bottom: 9px;
+  }
+
+  .stories-item__description {
+    font-size: 20px;
+    line-height: 25px;
+    margin-bottom: 35px;
+    min-height: 200px;
+  }
+
+  .stories-item__description-extended {
+    width: 100%;
+    display: grid;
+    grid-template-columns: 1.5fr 0.5fr 1fr;
+  }
+
+  .arrow {
+  	width: 70px;
+    display: flex;
+    align-items: center;
+  }
+
+  .stories-item__date {
+    font-size: 15px;
+    line-height: 14px;
+    font-weight: bold;
+    color: #d1d1d1;
+    opacity: 1;
+    text-align: right;
+  }
+
+  .gallery {
+    display: block;
+    height: 15%;
+    background: linear-gradient(2.6deg, #fff 62.5%, #000 63%);
+    margin-top: -5px;
+    width: 100%;
+    padding-bottom: 125px;
+  }
+
+  .gallery__title {
+    font-size: 30px;
+    line-height: 29px;
+    color: #fff;
+    opacity: 1;
+    text-align: right;
+    text-transform: uppercase;
+    font-weight: bold;
+    margin-bottom: 13px;
+    margin-right: 43px;
+  }
+
+  .gallery-content {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: 2;
+    margin-left: 43px;
+    margin-right: 43px;
+  }
+
+  .gallery-content__item {
+    display: block;
+    margin-bottom: -4px;
+  }
+
+  .gallery__photo {
+    width: 100%;
+  }
+
+  .footer {
+    width: 100%;
+  }
+
+  .footer__content {
+    max-width: 88%;
+    padding-top: 60px;
+  }
+
+  .footer__column-title {
+    font-size: 12px;
+    line-height: 11px;
+  }
+
+  .footer-column__nav-link {
+  	max-width: 155px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)